app/user.go

```diff
@@ -1302,12 +1302,16 @@ func (a *App) PermanentDeleteUser(user *model.User) *model.AppError {
 				continue
 			}
 
-			if res {
-				a.RemoveFile(info.Path)
-			} else {
-				mlog.Warn(fmt.Sprintf("Unable to remove file '%s': %s", info.Path, err))
+			if !res {
+				mlog.Warn(fmt.Sprintf("File '%s' not found", info.Path))
```

Comment:
Can we use the structured log features of mlog here? Example: https://github.com/mattermost/mattermost-server/blob/master/migrations/worker.go#L75

Reply:
@crspeller Sure, that sounds like a good idea. Implemented the change and rebased the PR.